test/test_examples_2d_part3.jl

```diff
@@ -0,0 +1,34 @@
+module TestExamples2DTestExamples2DPart3
```

Comment:
```suggestion
module TestExamples2DPart3
```
Same for the part2 file, I think part1 is ok

Reply:
Thanks!